h/templates/panels/navbar.html.jinja2

```diff
@@ -30,8 +29,7 @@
   </template>
   <div class="nav-bar__content">
     <a href="/" title="Hypothesis homepage" class="nav-bar__logo-container"><!--
-      !--><img alt="Hypothesis logo" class="nav-bar__logo"
-               src="{{ asset_url('images/logo.svg') }}"></a>
+      !-->{{ svg_icon('hypothesis-logo', 'nav-bar__logo') }}</a>
```

Comment:
The "Hypothesis logo" alt text is gone. The `<a>` that this `<img>` was on already has a `title`.